apps/xlayers/src/app/editor/code/editor-container/editor-container.component.ts

```diff
@@ -54,6 +54,11 @@ export class EditorContainerComponent implements OnInit {
     ctrl.focus();
   }
 
+  generateSvelte() {
+    this.codeSetting = this.codegen.generate(CodeGenKind.Svelte);
+    this.updateState();
+  }
+
```

Comment:
I think this function should be removed and the onChange on [L45](https://github.com/xlayers/xlayers/pull/397/files#L45) should be used.

Reply:
Done. Removed it because it was unused.